🌐 Add Chinese translation for `docs/zh/docs/tutorial/dependencies/dependencies-with-yield.md`

jaystone776 opened the pull request:
…ield.md

as title

jaystone776 committed:
Add Chinese translation for tutorial-dependencies dependencies-with-yield.md

as title

waynerv commented on docs/zh/docs/tutorial/dependencies/dependencies-with-yield.md:
```diff
@@ -0,0 +1,234 @@
+
+把依赖项的 `yield` 置于 `try`  代码块中，就可以接收到所有使用该依赖项时触发的异常。
+
+假设执行过程中的某行代码（这行代码可能在*路径操作或其他依赖项中*）导致了数据库事务的「回滚」，或导致了任何其他错误，使用 `try`  就可以在当前依赖项中触发异常。
```
```suggestion
假设执行过程中的某行代码（这行代码可能在*路径操作或其他依赖项中*）导致了数据库事务的「回滚」，或导致了任何其他错误，使用 `try` 就可以在当前依赖项中触发异常。
```

waynerv commented on docs/zh/docs/tutorial/dependencies/dependencies-with-yield.md:
```diff
@@ -0,0 +1,234 @@
+
+通过这种方式，**FastAPI** 允许依赖项中的任意对象（例如，数据库会话）都可以用于后台任务。
+
+后台任务是在响应发送*后*执行的任务。所以，此时抛出  HTTPException  没有意义，因为没有办法改变*已发送*的响应。
```
```suggestion
后台任务是在响应发送*后*执行的任务。所以，此时抛出 HTTPException 没有意义，因为没有办法改变*已发送*的响应。
```

waynerv commented on docs/zh/docs/tutorial/dependencies/dependencies-with-yield.md:
```diff
@@ -0,0 +1,234 @@
+
+!!! tip "提示"
+
+    不要在 `yield` 语句之后，而是要在该语句之前触发 `HTTPException` 等异常。
```
```suggestion
    在依赖项的 `yield` 语句之前抛出 `HTTPException` 等异常仍然是可行的，但不能在 `yield` 之后。
```

waynerv reviewed (approved):
Excellent works!

jaystone776 committed:
Update dependencies-with-yield.md

change 其他 to 其它
change 开发人员 to 开发者
change 「」 to bold for better reading experience

tiangolo committed:
Merge branch 'master' into Add-Chinese-translation-for-tutorial-dependencies-dependencies-with-yield.md

pre-commit-ci[bot] committed:
🎨 [pre-commit.ci] Auto format from pre-commit.com hooks

tiangolo commented:
📝 Docs preview for commit 979ade870b9b1272d65bb19697869a291db22e6a at: https://6499d1a2bac8cd0a1954b002--fastapi.netlify.app

alejsdev commented:
Hi! This one was already handled in https://github.com/tiangolo/fastapi/pull/10870. For now, I'll close this one, but thanks for the help @jaystone776 @waynerv 🚀
